setup.cfg

```diff
@@ -0,0 +1,2 @@
+[pycodestyle]
+max-line-length = 120
```

Comment:
I'm curious as to why you chose 120. Isn't 79||80 the norm?

Reply:
The platform uses 120:
https://github.com/edx/edx-platform/blob/master/setup.cfg#L38
due to modern editors/monitors having lots of screen real-estate.